### ISSUE TITLE
Please update the links on documentation page, pointing to the new location - moved to /src

### ISSUE THREAD

**Issue by EonosLovos** [tensorflow/tensorflow#66572]
Hello!

Thank you for all the awesome things you have done!

I was browsing tf.keras documentation pages, for example

https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dropout

and noticed that the pages do not find the lines in source code as expected, instead the 404 is being shown in the middle of github page where source code would be supposed to be.

in order for the links to work as expected, it seems that after branch number and keras another folder, /src should be added.

I mean:

https://github.com/keras-team/keras/tree/v3.3.3/keras/layers/regularization/dropout.py#L6-L76

becomes

https://github.com/keras-team/keras/blob/v3.3.3/keras/src/layers/regularization/dropout.py#L6-L76

h_ttps://github.com/keras-team/keras/blob/v3.3.3/keras/ **src/** layers/regularization/dropout.py#L6-L76

**Comment by sachinprasadhs** [comment]
@MarkDaoust , Could you please look into this. Thanks!

**Comment by MarkDaoust** [comment]
Fix inflight.

**Comment by sachinprasadhs** [comment]
@EonosLovos Here is the fix commit https://github.com/tensorflow/tensorflow/commit/ca59b909c82d254f3dd20bb2acef4485347f31bf
@MarkDaoust , When does the change reflect in the webpage?

**Comment by MarkDaoust** [comment]
Probably Thursday.

**Comment by tilakrayal** [comment]
@EonosLovos,
Could you please try to access the respective Keras page in the Tensorflow.org page, now it is able to re-direct to correct source code. Please check and confirm if it is working in your case.
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dropout
https://github.com/keras-team/keras/blob/v3.3.3/keras/src/layers/regularization/dropout.py#L6-L76

Thank you!

**Comment by MarkDaoust** [comment]
This is actually fixed.

**Comment by EonosLovos** [comment]
@tilakrayal @MarkDaoust  Yes I confirm, it works very well in my case! :))

I did enter the Tensorflow.org page, and indeed now it is able to re-direct to correct source code instead of 404 error. 
I do confirm that it is working in my case.

Three links in page when clicked opened the expected parts in TensorFlow code. The Dropout definition, the configuration and also the symcall. 

Thank you so much!!!
